explorer/client/src/index.tsx

```diff
@@ -14,6 +14,8 @@ import theme from './theme'
 JavascriptTimeAgo.locale(en)
 moment.defaultFormat = 'YYYY-MM-DD h:mm:ss A'
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
```

Comment:
Was there no easy fix for this?

Reply:
@HenryNguyen5 forgot to fix that, did it now. thanks for reminding